doc/source/cluster/kuberay/quickstart.ipynb

```diff
@@ -0,0 +1,616 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f34f1b75",
+   "metadata": {},
+   "source": [
+    "(kuberay-quickstart)=\n",
+    "\n",
+    "# Getting Started\n",
+    "\n",
+    "In this guide, we discuss how to manage and interact with Ray clusters on Kubernetes.\n",
+    "\n",
+    "You can download this guide as an executable Jupyter notebook by clicking the download button on the top right of the page.\n",
+    "\n",
+    "\n",
+    "## Preparation\n",
+    "\n",
+    "### Install the latest Ray release"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6dcd7d93",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install -U ray[default]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "656a0707",
+   "metadata": {},
+   "source": [
+    "See the [installation instructions](https://docs.ray.io/en/latest/ray-overview/installation.html) for more details. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c0933e2f",
+   "metadata": {},
+   "source": [
+    "### Install kubectl\n",
+    "\n",
+    "We will use kubectl to interact with Kubernetes. Find installation instructions at the [Kubernetes documentation](https://kubernetes.io/docs/tasks/tools/#kubectl).\n",
+    "\n",
+    "### Access a Kubernetes cluster\n",
+    "\n",
+    "We will need access to a Kubernetes cluster. There are two options:\n",
+    "1. Configure access to a remote Kubernetes cluster\n",
+    "**OR**\n",
+    "\n",
+    "2. Run the examples locally by [installing kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation). Start your [kind](https://kind.sigs.k8s.io/) cluster by running the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c764b3ad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! kind create cluster"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "278726e0",
+   "metadata": {},
+   "source": [
+    "To run the example in this guide, make sure your Kubernetes cluster (or local Kind cluster) can accomodate\n",
+    "additional resource requests of 3 CPU and 2Gi memory. \n",
```

Comment:
For kind, is this the case by default?

Reply:
Depends on your machine and the local docker setup.